app/assets/javascripts/TT_init.js

```diff
@@ -77,6 +83,8 @@ $(document).ready(function() {
     event.which == 32 && startCallback(event);
     // ESC key
     event.which == 27 && resetCallback(event);
+    // PAUSE key
+    event.which == 19 && pauseCallback(event);
```

Comment:
Maybe adding a `|| event.which == 80` would help, so the feature would be accessible through the "p" key.